scenarios.md

```diff
@@ -250,6 +250,40 @@ A weakness is discovered in a widely used cryptographic algorithm and a decision
 1. Key revocation, chain of trust, etc. must all work for the expected lifetime of a version of the client software while these changes are made.
 1. The actions that different parties need to perform must be clearly articulated, along with the result of not performing those actions.
 
+### Scenario #10: Specifying a trusted key
+
+A user may want to trust an artifact only if it is signed by a specific key. Maybe this is an internally created artifact with a known signing key. This key may be distributed using a trusted third party mechanism.
+
+1. The user obtains a trusted key for a particular artifact.
+1. The user downloads and verifies the artifact using Notary v2 and their known key.
+
+**Implications of this requirement**
+
+1. Users must be allowed to configure specific trusted keys for specific artifacts.
+
+### Scenario #11: Using a default trusted key
+
+If a user does not have a specific key for a given artifact, verified using a third party system, they will need to determine the trusted signing key(s) for an artifact using a secure default method.
```

Comment:
Having all of the information collected into one query makes sense, especially for moving signatures between registries, etc. For the requirements here I was trying to avoid too many implementation details, but I'll add something about updating the chain from root.

Reply:
@mnm678 it's not clear what "secure default method" means.

Reply:
How is this different from what we were discussing with the last scenario of configuring a key to trust for all artifacts? I think I'm missing the reasoning behind treating a "default" key as special, and not just a standard key that can be used for any artifact.

Reply:
One key aspect of Notary v2 is the decentralized model, where the registry stores signatures to artifacts, but key storage is deferred to key management systems. If we can include some chain of trust content in the signature, and it can define what root key could be used, greatness.